be/src/olap/rowset/rowset_meta.h

```diff
@@ -314,6 +314,12 @@ class RowsetMeta {
 
     void set_txn_expiration(int64_t expiration) { _rowset_meta_pb.set_txn_expiration(expiration); }
 
+    // `seg_file_size` MUST ordered by segment id
+    void set_segments_file_size(const std::vector<size_t>& seg_file_size);
```

Comment:
how to ensure 
> `seg_file_size` MUST ordered by segment id

should we check the order or sort it in `set_segments_file_size`